rpcclient/examples/bitcoincorehttpbulk/main.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2014-2019 The btcsuite developers
```

Comment:
tbh - I think it might make sense to do this in a separate PR. The copyrights are kind of a mess:

![image](https://user-images.githubusercontent.com/6392429/100785976-7181fd00-33df-11eb-83e9-8b8a50858abd.png)


![image](https://user-images.githubusercontent.com/6392429/100785986-75ae1a80-33df-11eb-8346-50399bff6a53.png)

![image](https://user-images.githubusercontent.com/6392429/100785995-7a72ce80-33df-11eb-9071-5073f28a709e.png)


I'm not sure if that's because these were seperate reops that got merged together. @jcvernaleo might be more familiar with the history but it looks like it's just messy rather than any nuance: https://github.com/btcsuite/fastsha256/pull/6#issuecomment-239336912